midend/simplifyKey.h

```diff
@@ -23,15 +23,19 @@ limitations under the License.
 
 namespace P4 {
 
-// Policy used to decide whether a key expression is too complex.
+/**
+ * Policy used to decide whether a key expression is too complex.
+ */
 class KeyIsComplex {
  public:
     virtual ~KeyIsComplex() {}
     virtual bool isTooComplex(const IR::Expression* expression) const = 0;
 };
 
-// Policy which returns 'true' whenever a key is not a left value
-// or a call to the isValid().
+/**
+ * Policy which returns 'true' whenever a key is not a left value
```

Comment:
Instead of saying "returns" how about a different verb? For example "Policy that tracks whether a key is..."